src/transformers/pipelines/zero_shot_classification.py

```diff
@@ -113,13 +113,31 @@ def _parse_and_tokenize(
                 )
                 inputs.append(model_input)
         else:
-            inputs = self.tokenizer(
-                sequence_pairs,
-                add_special_tokens=add_special_tokens,
-                return_tensors=return_tensors,
-                padding=padding,
-                truncation=truncation,
-            )
+            try:
+                inputs = self.tokenizer(
+                    sequence_pairs,
+                    add_special_tokens=add_special_tokens,
+                    return_tensors=return_tensors,
+                    padding=padding,
+                    truncation=truncation,
+                )
+            except Exception as e:
+                if "too short" in str(e):
+                    # tokenizers might yell that we want to truncate
+                    # to a value that is not even reached by the input.
+                    # In that case we don't want to truncate.
+                    # It seems there's not a really better way to catch that
+                    # exception.
```

Comment:
Could you provide a reproducer for this issue? I can't seem to find where that would come from

Reply:
Basic test for the pipelines `RUN_PIPELINE_TESTS=1 pytest -sv tests/test_pipelines_zero_shot_classification.py` will do (on the LED test)